core/src/Kokkos_Graph.hpp

```diff
@@ -148,10 +148,20 @@ Graph<ExecutionSpace> create_graph(ExecutionSpace ex, Closure&& arg_closure) {
   return rv;
 }
 
+template <class ExecutionSpace = DefaultExecutionSpace>
+std::enable_if_t<Kokkos::is_execution_space_v<ExecutionSpace>,
+                 Graph<ExecutionSpace>>
+create_graph(const ExecutionSpace& exec = ExecutionSpace{}) {
+  return create_graph(exec, [](const auto& /* root */) {});
+}
```

Comment:
I need to fix this. For now, that would put the use in a situation wherein we've got 2 root nodes, but only one is connected to the user's nodes. The other one is useless.

Reply:
Done.